docker-compose-production.yml

```diff
@@ -4,7 +4,7 @@ version: '3'
 
 services:
   telescope:
-    container_name: telescope_staging
+    container_name: telescope_production
```

Comment:
Do we need to update https://github.com/Seneca-CDOT/telescope/blob/d080857cb959687b0182b11d7e630b540ac24dee/nginx-staging.conf to use this new name?  Or is that separate?

Reply:
Oh, yes, my bad.